run-integration-tests.py

```diff
@@ -245,21 +276,22 @@ def __exit__(self, tpe, value, traceback):
         action="store_true",
         help="Use testpypi for testing pip installation")
     parser.add_argument(
-        "--packages",
+        "--run-storage-dynamodb-integration-tests",
         required=False,
-        default=None,
-        help="Additional packages required for some tests")
+        default=False,
+        action="store_true",
+        help="Run the DynamoDB integration tests (and only them)")
     parser.add_argument(
-        "--jars",
+        "--packages",
```

Comment:
idea: `--dbb-packages` ? and `--dbb-conf` below? to make it crystal clear this is only for dynamodb integration tests?

just an idea. im easy.

Reply:
yeah I like this